var/spack/repos/builtin/packages/racon/package.py

```diff
@@ -11,26 +11,28 @@ class Racon(CMakePackage):
     """Ultrafast consensus module for raw de novo genome assembly of long
     uncorrected reads."""
 
-    homepage = "https://github.com/isovic/racon"
-    url = "https://github.com/isovic/racon/releases/download/1.2.1/racon-v1.2.1.tar.gz"
-
-    version("1.4.3", sha256="dfce0bae8234c414ef72b690247701b4299e39a2593bcda548a7a864f51de7f2")
```

Comment:
Would be preferable to first deprecate these old versions and remove them after the next minor release so that this doesn't come as a surprise to any users. You can add a `url_for_version` that chooses the appropriate url to use.